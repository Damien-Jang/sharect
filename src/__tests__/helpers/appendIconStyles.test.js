import {
  appendIconStyle,
  appendMobileIconStyle
} from '../../helpers/appendIconStyles'

const iconColor = '#F34'
const mobileIconSize = 5

describe('appendIconStyle', () => {
  it('should create a style tag', () => {
    appendIconStyle({ iconColor })

    expect(document.body).toMatchSnapshot()
  })
})

describe('appendMobileIconStyle', () => {
  it('should update the style for mobile', () => {
    appendMobileIconStyle(iconColor, mobileIconSize)

    expect(document.body).toMatchSnapshot()
  })
  
})

