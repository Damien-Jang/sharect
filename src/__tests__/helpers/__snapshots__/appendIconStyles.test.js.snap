// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`appendIconStyle should create a style tag 1`] = `
<body>
  <style
    id="sharect-style"
  >
    .sharect svg{fill:#F34;}
  </style>
</body>
`;

exports[`appendMobileIconStyle should update the style for mobile 1`] = `
<body>
  <style
    id="sharect-style"
  >
    .sharect svg{fill:#F34;width:5px;height:5px;}
  </style>
</body>
`;
